Tools/machines/lxplus-cern/spack.yaml

```
@@ -0,0 +1,47 @@
+spack:
+# CUDA
+  definitions:
+  - cuda: []
+  - cuda: [cuda ^ncurses+symlinks, blaspp +cuda ^ncurses+symlinks]
+    when: env.get("SPACK_STACK_USE_CUDA", "") == "1"
+
+# basic Python dependencies for WarpX Python PICMI bindings
+  - python: []
+  - python: [python ^ncurses+symlinks, py-pip ^ncurses+symlinks, py-mpi4py ^ncurses+symlinks, py-numpy ^ncurses+symlinks,
+      py-periodictable ^ncurses+symlinks, py-picmistandard ^ncurses+symlinks]
+    when: env.get("SPACK_STACK_USE_PYTHON", "") == "1"
+  specs:
+  - ncurses+symlinks
+  - adios2 ^ncurses+symlinks
+  - blaspp ^ncurses+symlinks
+  - ccache ^ncurses+symlinks
+  - cmake ^ncurses+symlinks
+  - fftw ^ncurses+symlinks
+  - hdf5 ^ncurses+symlinks
+  - lapackpp ^ncurses+symlinks
+  - mpi ^ncurses+symlinks
+  - openpmd-api ^ncurses+symlinks
```

The nice thing about this env: the `ncurses +symlinks` constrain in it will be fulfilled now by the whole concretization :)

Btw, did you add this because you saw problems otherwise with a package?

```suggestion
  - adios2
  - blaspp
  - ccache
  - cmake
  - fftw
  - hdf5
  - lapackpp
  - mpi
  - openpmd-api
```